app/models/policy_host.rb

```diff
@@ -8,10 +8,24 @@ class PolicyHost < ApplicationRecord
   validates :policy, presence: true
   validates :host, presence: true, on: :create
   validates :host_id, presence: true, uniqueness: { scope: :policy }
+  validate :host_supported?
```

Comment:
I would move this into the `host` validation with a custom lambda because:
* it semantically belongs there and should error on it
* it should be validated only when creating (we should not break existing policies by preventing to update them)
* less lines :wink: